### DISCUSSION TITLE
Welcome to geo-deep-learning Discussions!

### DISCUSSION THREAD

**Discussion by remtav** [NRCan/geo-deep-learning#170] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**********************

Hey! There is a new Discussion feature in Github! No more need to systematically open an issue to talk about the real stuff (aka GDL)

**Comment by mkutu** [comment]
How can I help?

**Reply by ymoisan** [reply]
Hey @mkutu , I was just wondering who you were :-).  I see you have opened a PR.  Thanx !

Just so you know, we are onboarding new staff this week that will be working with us on GDL.  We expect a lot more to happen in GDL in the following weeks, so I think to answer your question we (the dev team) need to identify our objectives and milestones first.  Then it should be clearer how you could help.  We'll keep you posted.

Cheers!

**Reply by mkutu** [reply]
Hi @ymoisan. My name is Matthew Kutugata and I'm a [graduate student at Texas A&M.](https://soilcrop.tamu.edu/people/kutugata-matthew/) I'm entering my second semester and my research is at the intersection of remote sensing, computer vision, and sustainable ag. 

I'll be honest, that was my first PR and I thank @CharlesAuthier for taking the time to give detailed comments. I know maintaining a project like GDL and reviewing multiple PRs can be very time consuming. I hope to be a little more organized on my next PR where I've included an additional model (FCN). 

It's great to hear that GDL will be getting more attention. Thank you and your team for building it. It is quite impressive and I'm excited to use it. This is the first example I've seen that logically includes tiff metadata in a dataloader. Also, I think the interchangeability of models will come in very handy for addressing problems in different types of agroecosystems.  

Yes, please keep me posted. I'm happy to talk more about what I can do.  For what it's worth, I think GDL is a powerful tool with a lot of potential.

cheers
